docs/ui/components/Form/Checkbox.tsx

```diff
@@ -1,15 +1,15 @@
 import { mergeClasses } from '@expo/styleguide';
 import { CheckIcon } from '@expo/styleguide-icons/outline/CheckIcon';
 import { forwardRef, Ref, InputHTMLAttributes, ReactNode } from 'react';
-
 import { P } from '../Text';
 
 type Props = InputHTMLAttributes<HTMLInputElement> & {
   label?: ReactNode;
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void; // Ensure onChange is included
```

Comment:
```suggestion
  onChange?: (event: ChangeEvent<HTMLInputElement>) => void;
```

Few nits, comment feels redundant, since code is self-explonatory.